widget/task_preview.lua

```diff
@@ -24,99 +27,118 @@ local function draw_widget(c, task_preview_box, screen_radius, widget_bg,
     cr.operator = cairo.Operator.SOURCE
     cr:paint()
 
-    task_preview_box:setup{
-        {
+    local widget = wibox.widget{
+        widget_template or {
```

Comment:
I think it would be better to merge incoming table with the default one

Reply:
wdym?